--- a/dmenu.c	2015-05-04 13:25:48.700960568 -0400
+++ b/dmenu.c	2015-05-04 13:27:38.441287455 -0400
@@ -607,6 +607,12 @@
 	                    DefaultVisual(dc->dpy, screen),
 	                    CWOverrideRedirect | CWBackPixel | CWEventMask, &swa);
 
+    XClassHint *hint = XAllocClassHint();
+    hint->res_name = "dmenu";                                              
+    hint->res_class = "dmenu";                                       
+    XSetClassHint(dc->dpy, win, hint);
+    XFree(hint);
+
 	/* open input methods */
 	xim = XOpenIM(dc->dpy, NULL, NULL, NULL);
 	xic = XCreateIC(xim, XNInputStyle, XIMPreeditNothing | XIMStatusNothing,
