--- a/dmenu.c	2015-05-04 15:35:24.653545625 -0400
+++ b/dmenu.c	2015-05-04 15:34:48.934335932 -0400
@@ -298,6 +298,15 @@
 		default:
 			return;
 		}
+	else if(ev->state & ShiftMask)
+		switch(ksym) {
+		case XK_Insert:
+			XConvertSelection(dc->dpy, XA_PRIMARY,
+			                  utf8, utf8, win, CurrentTime);
+			return;
+		default:
+			return;
+}
 	switch(ksym) {
 	default:
 		if(!iscntrl(*buf))
