diff --git a/config.def.h b/config.def.h
index 513c734..508beaf 100644
--- a/config.def.h
+++ b/config.def.h
@@ -80,6 +80,7 @@ static Layout layouts[] = {
 static KeyBinding bindings[] = {
 	{ { MOD, 'c',          }, { create,         { NULL }                    } },
 	{ { MOD, 'C',          }, { create,         { NULL, NULL, "$CWD" }      } },
+	{ { MOD, 'r',          }, { create,         { "plumb", NULL, "$CWD" }   } }, //plumb
 	{ { MOD, 'x', 'x',     }, { killclient,     { NULL }                    } },
 	{ { MOD, 'j',          }, { focusnext,      { NULL }                    } },
 	{ { MOD, 'J',          }, { focusdown,      { NULL }                    } },
