--- src/slock-1.2/slock.c	2014-12-22 05:18:07.000000000 -0500
+++ /home/Link/slock/slockclass	2015-05-04 20:29:26.255732131 -0400
@@ -220,6 +220,13 @@
 	invisible = XCreatePixmapCursor(dpy, lock->pmap, lock->pmap, &color, &color, 0, 0);
 	XDefineCursor(dpy, lock->win, invisible);
 	XMapRaised(dpy, lock->win);
+
+	XClassHint *hint = XAllocClassHint(); 
+	hint->res_name = "slock";
+	hint->res_class = "slock";
+	XSetClassHint(dpy, lock->win, hint);
+	XFree(hint);
+
 	for(len = 1000; len; len--) {
 		if(XGrabPointer(dpy, lock->root, False, ButtonPressMask | ButtonReleaseMask | PointerMotionMask,
 			GrabModeAsync, GrabModeAsync, None, invisible, CurrentTime) == GrabSuccess)
