--- dwm.c_	2018-05-28 03:08:38.707219351 -0400
+++ dwm.c	2018-05-28 03:10:42.540553642 -0400
@@ -2171,6 +2171,12 @@
 		die("pledge");
 #endif /* __OpenBSD__ */
 	scan();
+	setenv("DMENU_FONT", dmenufont, 1);
+	setenv("DMENU_COL_GRAY1", col_gray1, 1);
+	setenv("DMENU_COL_GRAY2", col_gray2, 1);
+	setenv("DMENU_COL_GRAY3", col_gray3, 1);
+	setenv("DMENU_COL_GRAY4", col_gray4, 1);
+	setenv("DMENU_COL_CYAN", col_cyan, 1);
 	run();
 	cleanup();
 	XCloseDisplay(dpy);
