diff --git a/config.h b/config.h
index b18dfe5..5e56399 100644
--- a/config.h
+++ b/config.h
@@ -20,7 +20,7 @@ static const char *colors[][3]      = {
 };
 
 /* tagging */
-static const char *tags[] = { "cmd", "www", "irc", "dvr", "gog", "etc" };
+static const char *tags[] = { "etc" };
 
 static const Rule rules[] = {
 	/* xprop(1):
@@ -28,29 +28,6 @@ static const Rule rules[] = {
 	 *	WM_NAME(STRING) = title
 	 */
 	/* class,           instance,                                title,      tags mask,  isfloating,  monitor */
-//cmd
-	{ "st-256color",    "st-256color",                           NULL,       1,          False,       -1 },
-//www
-	{ "Chromium",       "chromium",                              NULL,       1 << 1,     False,       -1 },
-	{ "Chromium",       "crx_pfpeapihoiogbcmdmnibeplnikfnhoge",  NULL,       1 << 1,     False,       -1 },
-	{ "Firefox",        NULL,                                    NULL,       1 << 1,     False,       -1 },
-	{ "qutebrowser",    "qutebrowser",                           NULL,       1 << 1,     False,       -1 },
-//irc
-	{ "ircterm",        NULL,                                    NULL,       1 << 2,     False,       -1 },
-	{ "skypeforlinux",  "skypeforlinux",                         NULL,       1 << 2,     False,       -1 },
-//dvr
-	{ "Chromium",       "crx_deceagebecbceejblnlcjooeohmmeldh",  NULL,       1 << 3,     False,       -1 },
-	{ "Chromium",       "crx_fpniocchabmgenibceglhnfeimmdhdfm",  NULL,       1 << 3,     False,       -1 },
-	{ "plexmediaplayer","plexmediaplayer",                       NULL,       1 << 3,     False,       -1 },
-//gog
-	{ "steam",          "steam",                                 NULL,       1 << 4,     False,       -1 },       // Big Picture Mode
-	{ "Steam",          "Steam",                                 NULL,       1 << 4,     False,       -1 },
-	{ "Steam",          "Steam",                                 "Friends",  1 << 4,     False,       -1 },
-//etc
-	{ "Client.pyw",     "client.pyw",                            NULL,       1 << 5,     False,       -1 },
-	{ "xfreerdp",       "xfreerdp",                              NULL,       1 << 5,     False,       -1 },
-	{ "Okular",         "okular",                                NULL,       1 << 5,     False,       -1 },
-	{ "Deluge",         "deluge",                                NULL,       1 << 5,     False,       -1 },
 //all
 	{ "popterm",        NULL,                                    NULL,       ~0,         True,        -1 },
 	{ "qutebrowser",    "popwww",                                NULL,       ~0,         True,        -1 },
