diff --git a/config.h b/config.h
index 5e56399..cf3694d 100644
--- a/config.h
+++ b/config.h
@@ -29,6 +29,7 @@ static const Rule rules[] = {
 	 */
 	/* class,           instance,                                title,      tags mask,  isfloating,  monitor */
 //all
+ 	{ NULL,            "foobar2000.exe",                         NULL,       ~0,         False,        2 },
 	{ "popterm",        NULL,                                    NULL,       ~0,         True,        -1 },
 	{ "qutebrowser",    "popwww",                                NULL,       ~0,         True,        -1 },
 };
@@ -75,6 +76,9 @@ static const char *clipmenu[]    = { "clipmenu",   "-m", dmenumon, "-fn", dmenuf
 static const char *j4dmenu[]     = { "fondler.sh", "j4", dmenumon, NULL };
 static const char *brightdown[]  = { "fondler.sh", "brightdown", NULL };
 static const char *brightup[]    = { "fondler.sh", "brightup", NULL };
+static const char *foobnext[]    = { "fondler.sh", "foobnext", NULL };
+static const char *foobplay[]    = { "fondler.sh", "foobplay", NULL };
+static const char *foobprev[]    = { "fondler.sh", "foobprev", NULL };
 static const char *lock[]        = { "fondler.sh", "lock", NULL };
 static const char *popterm[]     = { "popterm", "abduco", "-A", "popterm", NULL };
 static const char *popwww[]      = { "popwww", NULL };
@@ -99,6 +103,9 @@ static Key keys[] = {
 	{ 0,                   XF86XK_AudioMute,          spawn,           {.v = mute } },
 	{ 0,                   XF86XK_AudioLowerVolume,   spawn,           {.v = voldown } },
 	{ 0,                   XF86XK_AudioRaiseVolume,   spawn,           {.v = volup } },
+	{ 0,                   XF86XK_AudioNext,          spawn,           {.v = foobnext } },
+	{ 0,                   XF86XK_AudioPlay,          spawn,           {.v = foobplay } },
+	{ 0,                   XF86XK_AudioPrev,          spawn,           {.v = foobprev } },
 	{ 0,                   XF86XK_Display,            spawn,           {.v = rotate } },
 	{ 0,                   XF86XK_MonBrightnessDown,  spawn,           {.v = brightdown } },
 	{ 0,                   XF86XK_MonBrightnessUp,    spawn,           {.v = brightup } },
