--- a/dvtm-status	2015-02-19 11:51:23.000000000 -0500
+++ b/dvtm-status	2015-09-06 18:35:29.909155344 -0400
@@ -6,8 +6,31 @@
 chmod 600 $FIFO
 
 while true; do
-	date +%H:%M
-	sleep 60
+        # Power Status
+        if [ "$(cat /sys/class/power_supply/*/online)" -eq "1" ]; then
+                POWERSOURCE="+";
+        else
+                POWERSOURCE="-";
+        fi;
+
+        # Battery Status
+        BATTERYLEVEL=[$(cat /sys/class/power_supply/BAT0/capacity)];
+        if [ -d /sys/class/power_supply/BAT1 ]; then
+                SPAREBATTERYLEVEL=[$(cat /sys/class/power_supply/BAT1/capacity)];
+        fi;
+
+        # Network Status
+        NETPROFILE="$(netctl list | sed -e '/\*/!d' -e 's/^\* //') |";
+
+        # Date and Time
+        CLOCK=$( date '+%H:%M' );
+
+        # Load
+        LOAD="`cat /proc/loadavg | awk '{print $1, $2, $3}'`";
+
+        # Overall output command
+        echo "$LOAD | $NETPROFILE $SPAREBATTERYLEVEL[$POWERSOURCE]$BATTERYLEVEL | $CLOCK";
+        sleep 60
 done > $FIFO &
 
 STATUS_PID=$!
