--- a/dwm.c	2015-05-04 14:16:23.926513366 -0400
+++ b/dwm.c	2015-05-04 14:23:40.662047887 -0400
@@ -1738,6 +1738,9 @@
 		.background_pixmap = ParentRelative,
 		.event_mask = ButtonPressMask|ExposureMask
 	};
+	XClassHint *ch = XAllocClassHint();
+	ch->res_name = "dwmstatus";
+	ch->res_class = "dwm";
 	for(m = mons; m; m = m->next) {
 		if (m->barwin)
 			continue;
@@ -1746,7 +1749,9 @@
 		                          CWOverrideRedirect|CWBackPixmap|CWEventMask, &wa);
 		XDefineCursor(dpy, m->barwin, cursor[CurNormal]->cursor);
 		XMapRaised(dpy, m->barwin);
+		XSetClassHint(dpy, m->barwin, ch);
 	}
+	XFree(ch);
 }
 
 void
