--- a/config.h	2016-03-09 20:57:54.583585425 -0500
+++ b/config.h	2016-03-09 21:11:39.486955937 -0500
@@ -86,6 +86,9 @@
 static const char *j4dmenu[]    = { "fondler.sh", "-m", dmenumon, "-fn", dmenufont, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, "-i", NULL };
 static const char *brightdown[] = { "fondler.sh", "brightdown", NULL };
 static const char *brightup[]   = { "fondler.sh", "brightup", NULL };
+static const char *foobnext[]   = { "wine", "~/.foobar2000/foobar2000.exe", "/next", NULL };
+static const char *foobplay[]   = { "wine", "~/.foobar2000/foobar2000.exe", "/playpause", NULL };
+static const char *foobprev[]   = { "wine", "~/.foobar2000/foobar2000.exe", "/prev", NULL };
 static const char *lock[]       = { "fondler.sh", "lock", NULL };
 static const char *mute[]       = { "amixer", "-q", "sset", "Master", "toggle", NULL };
 static const char *newbg[]      = { "systemctl", "--user", "start", "feh-wallpaper.service", "feh-wallpaper.timer", NULL };
@@ -103,6 +106,9 @@
 	{ 0,                  XF86XK_AudioMute,         spawn,          {.v = mute } },
 	{ 0,                  XF86XK_AudioLowerVolume,  spawn,          {.v = voldown } },
 	{ 0,                  XF86XK_AudioRaiseVolume,  spawn,          {.v = volup } },
+	{ 0,                  XF86XK_AudioNext,         spawn,          {.v = foobnext } },
+	{ 0,                  XF86XK_AudioPlay,         spawn,          {.v = foobplay } },
+	{ 0,                  XF86XK_AudioPrev,         spawn,          {.v = foobprev } },
 	{ 0,                  XF86XK_Display,           spawn,          {.v = rotate } },
 	{ 0,                  XF86XK_MonBrightnessDown, spawn,          {.v = brightdown } },
 	{ 0,                  XF86XK_MonBrightnessUp,   spawn,          {.v = brightup } },
