--- config.h	2016-03-23 23:53:38.924557426 -0400
+++ config.h.new	2016-03-24 00:07:24.304591866 -0400
@@ -45,6 +45,7 @@
 	//all
 	{ "mpv",         NULL,          NULL,      ~0,            True,        -1 },
 	{ "Sxiv",        NULL,          NULL,      ~0,            True,        -1 },
+	{ NULL,       "foobar2000.exe", NULL,      ~0,            False,        2 },
 };
 
 /* layout(s) */
@@ -87,6 +88,9 @@
 static const char *j4dmenu[]    = { "fondler.sh", "-m", dmenumon, "-fn", dmenufont, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, "-i", NULL };
 static const char *brightdown[] = { "fondler.sh", "brightdown", NULL };
 static const char *brightup[]   = { "fondler.sh", "brightup", NULL };
+static const char *foobnext[]   = { "fondler.sh", "foobnext", NULL };
+static const char *foobplay[]   = { "fondler.sh", "foobplay", NULL };
+static const char *foobprev[]   = { "fondler.sh", "foobprev", NULL };
 static const char *dunsttoggle[]= { "fondler.sh", "dunsttoggle", NULL };
 static const char *lock[]       = { "fondler.sh", "lock", NULL };
 static const char *mute[]       = { "amixer", "-q", "sset", "Master", "toggle", NULL };
@@ -105,6 +109,9 @@
 	{ 0,                  XF86XK_AudioMute,         spawn,          {.v = mute } },
 	{ 0,                  XF86XK_AudioLowerVolume,  spawn,          {.v = voldown } },
 	{ 0,                  XF86XK_AudioRaiseVolume,  spawn,          {.v = volup } },
+	{ 0,                  XF86XK_AudioNext,         spawn,          {.v = foobnext } },
+	{ 0,                  XF86XK_AudioPlay,         spawn,          {.v = foobplay } },
+	{ 0,                  XF86XK_AudioPrev,         spawn,          {.v = foobprev } },
 	{ 0,                  XF86XK_Battery,           spawn,          {.v = dunsttoggle } },
 	{ 0,                  XF86XK_Display,           spawn,          {.v = rotate } },
 	{ 0,                  XF86XK_MonBrightnessDown, spawn,          {.v = brightdown } },
